mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -409,6 +409,22 @@ func schemaNodeConfig() *schema.Schema {
 					ForceNew: true,
 					Description: `Setting this field will assign instances of this pool to run on the specified node group. This is useful for running workloads on sole tenant nodes.`,
 				},
+				"advanced_machine_features": {
+					Type:        schema.TypeList,
+					MaxItems:    1,
```

Comment:
I think this one should be `ForceNew` as well. There are some edge cases where it could come up.